main.js

```diff
@@ -0,0 +1,179 @@
+// Exerciess 1: in một bảng của một số bất kì với các số từ 1 đến 10 và hiển thị ra kết quả 
+const multiplicationTable = num =>{
+    if (num < 1 || num > 10) return ;
+
+    console.log('Bảng tính nhân của:', num, 'là');
+    for (let i = 1; i <= 10; i++) {
+        return console.log( num, 'x', i,'=', i * num);
+    }
+}
+
+/**
+ * Exerciess 2:
+ * Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 
+ * In các số chẵn từ 1 đến n Vd n = 6: in ra: 2 4 6
+ */
+const findEvenNumber = (num) => {
+    if ( 1 <= num <= 30) {
+        for (let i = 1; i <= num; i++) { 
+            if (i % 2 === 0)
+            console.log(i);
+        }
+    }    
+}
+
+/**
+ * Exerciess 3: 
+ * Hàm nhận tham số là n (số nguyên dương), 1 <= n <= 30 Tính tổng từ 1 đến n
+ * vd nhập vào 6: Tổng = 1 + 2 + 3 + 4 + 5 + 6 = 21
+ */
+
+const sum = num => {
+    if (num < 1 || num > 30) return 'Hãy điền 1 số từ 1 đến 30'
+
+    let sum = 0
+    if ( 1 <= num <= 30 ) {
```

Comment:
Bên trên đã kiểm tra nếu `num < 1` hoặc `num > 30` thì dừng
Thì xuống này sao phải kiểm tra num có nằm trong `1-30` hay ko nữa?

Và trong máy tính ko có phép toán như viết tay thế này:
`1 <= num <= 30` chứng tỏ viết code nhưng không hề chạy thử
Phải viết là: `num >= 1 && num <= 30`

Và kể cả biểu thức `1 <= num <= 30` là đúng thì dòng `if` phải viết xóa 2 dấu cách đi:
```suggestion
    if (1 <= num <= 30) {
```

Reply:
em chạy thử nó có ra mà anh